### PR TITLE
Set up architecture to allow load testing of the Prime API in Experimental

### PR DESCRIPTION
**Pull request by sandy-wright**
## Description

The focus of this PR was to set up a docker container that can run load testing against the MilMove experimental environment. To accomplish this, several tasks had to be completed:

- `chamber` needed to be set up with a new service that holds the load testing secrets, namely the experimental certificate and private key. This involved adding a couple `.envrc` files so that `direnv allow` can load these secrets.

- the `MilMoveHostMixin` class needed to be updated so that it could determine which TLS credentials to use based off the environment. This was the most complicated part of the update, as it involves creating a temporary cert/key file for experimental certs, then removing said file once the test has completed.

  I also renamed the `utils/mixins.py` file to `util/hosts.py` and moved some other host-related code into this file, which hopefully made organization a little more explicit. 
  
- `docker-compose.yaml` was written to test the experimental environment by default.

Any other changes are miscellaneous formatting updates so that our code agrees with all of the linters.

## Reviewer Notes

The only Prime API endpoint that works in experimental currently is `fetchMTOUpdates`. No other request will succeed, and that's okay for right now.

Also, I will be updating the CircleCI config in a separate PR to lighten the load on this one at least a tiny bit. 

## Setup

- Run these commands in sequence:
  ```sh
  direnv allow  <- make sure you use your AWS GovCloud OTP
  docker-compose build
  docker-compose up
  ```
- Open <http://localhost:8089/>. 
- Choose any number of users and any spawn rate. Make sure the host is still "exp". Then click "Start swarmming".
- Watch the results. You should eventually see some 200 responses to the `/move-task-orders` path. Yay, we did it!
- ~If you aren't seeing any results, it's likely because you are unlucky and these locust users keep trying all the tasks that can't be run yet, instead of `fetch_mto_updates`. To fix this:~
  - ~Go into the `docker-compose.yaml` file and add `--tags fetchMTOUpdates` to the end of the `command`,~
  - ~Re-run `docker-compose up`, and~
  - ~Start a new load test from the locust interface.~
- EDIT: I went ahead and changed the default command to have `--tags fetchMTOUpdates`, which we will remove when we fix the other endpoints.

## Code Review Verification Steps

* [x] Request review from a member of a different team.
* [x] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-6237) for this change.

### CODE REVIEW

**Review by abbyoung — APPROVED** [review]
works great, this is awesome work!

File: .gitignore [hunk 1]
```diff
@@ -12,4 +13,10 @@
...
 /venv/*
-.python-version
\ No newline at end of file
+.python-version
+
+# Envrc
+.envrc.chamber
+.envrc.local*
+exp_*.pem
+stg_*.pem
```
Comment (mdawn) [inline .gitignore:22]: why is this in here?
did you use a staging cert?

File: README.md [hunk 2]
```diff
@@ -326,7 +328,7 @@ commands include:
 
 Each of these commands opens the Locust interface for initiating and monitoring the tests, set to [http://localhost:8089](http://localhost:8089).
 Using this interface, you can set the number of users to simulate and their hatch rate, then start and stop the test at
-will. For the host, you can enter a full URL address, or you can simply enter "local", "staging", or "experimental", and
+will. For the host, you can enter a full URL address, or you can simply enter "local", "stg" (for staging), or "exp" (for experimental), and
```
Comment (mdawn) [inline README.md:331]: If we're not ever going to hit staging, this & other refs to it can probably come out.

**Review by mdawn — COMMENTED** [review]
🚀  AWESOME!!!

File: README.md (hunk 2 above)
Comment (sandy-wright) [reply README.md:331]: yeah, it's confusing. I had it beat into me last time that the goal was to test in staging, but is **in the past**, so I'll remove all these refs

File: .gitignore (hunk 1 above)
Comment (sandy-wright) [reply .gitignore:22]: no no, just subconsciously taking it into account. I'll remove it

**Pull request merged**